NewEvaluation.html

```diff
@@ -23,56 +23,62 @@
 
 
 	<div class = "mainBlock">
-
+	<div class="spacing"></div>
 		<form >
-			<input type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
-			<input type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
-			<input type="date" name="Date" placeholder="dd/mm/yyyy" required/>
+			<input class="input-candidate" type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
+			<input class="input-candidate" type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
+			<input class="input-candidate" type="date" name="Date" placeholder="dd/mm/yyyy" required/>
 		</form>
 
+		<div class="littleSpacing"></div>
```

Comment:
Ahh, the notorious artificially introduced HTML element used for styling! A good separation of concerns would have the styling offloaded to the CSS layer, while the HTML is kept clean, storing only the data to be displayed, and not information on how to display that data.